lib/widgets/forms/plan_form.dart

```diff
@@ -0,0 +1,372 @@
+				meta: (index, item) => item
+			),
+			isTwoLine: true,
+			choiceType: SmartSelectChoiceType.chips,
+			choiceConfig: SmartSelectChoiceConfig(
+				builder: (item, checked, onChange) => 
+					Theme(
+						data: ThemeData(textTheme: Theme.of(context).textTheme),
+						child: ItemCard(
+							title: item.title,
+							subtitle: AppLocales.of(context).translate(checked ? 'actions.selected' : 'actions.tapToSelect'),
+							graphicType: GraphicAssetType.childAvatars,
+							graphic: item.meta.avatar,
+							graphicShowCheckmark: checked,
+							graphicHeight: 44.0,
+							onTapped: onChange != null ? () => onChange(item.value, !checked) : null,
+							isActive: checked
+						)
+					)
+			),
+			modalType: SmartSelectModalType.bottomSheet,
+			modalConfig: SmartSelectModalConfig(
+				searchBarHint: AppLocales.of(context).translate('actions.search'),
+				trailing: buildBottomSheetBar(context, children.length > 1, children, widget.plan.children)
+			),
+			leading: Padding(padding: EdgeInsets.all(8.0), child: Icon(Icons.people)),
+			onChange: (val) => setState(() {
+				FocusManager.instance.primaryFocus.unfocus();
+				widget.plan.children.clear();
+				widget.plan.children = val;
+			})
+		);
+	}
+
+	Widget buildBottomSheetBarButton(Color color, IconData icon, String text, Function onPressed) {
+		return Expanded(
+			child: RaisedButton(
+				shape: ContinuousRectangleBorder(),
+				padding: EdgeInsets.symmetric(vertical: 12.0),
+				color: color,
+				child: Row(
+					mainAxisAlignment: MainAxisAlignment.center,
+					children: <Widget>[
+						Padding(
+							padding: EdgeInsets.only(right: AppBoxProperties.buttonIconPadding),
+							child: Icon(icon, color: Colors.white)
+						),
+						Text(text, style: TextStyle(fontWeight: FontWeight.normal, color: Colors.white))
+					]
+				),
+				materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+				onPressed: onPressed
+			)
+		);
+	}
+
+	Widget buildBottomSheetBar(BuildContext context, bool selectAllButton, List<dynamic> sourceList, List<dynamic> valueList) {
+		/* SELECT ALL
+		Function onSelectAll = () {
+			if(sourceList.length == valueList.length) {
+				setState(() => valueList.clear());
+			} else {
+				setState(() => valueList.clear());
+				List<Mongo.ObjectId> allChildren = List<Mongo.ObjectId>();
+				sourceList.forEach((element) {
+					allChildren.add(element.id);
+				});
+				setState(() => valueList = allChildren);
+			}
+		};
+		*/
+		return Row(
+			mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+			children: [
+				/* SELECT ALL
+				if(selectAllButton)
+					if(sourceList.length == valueList.length)
+						buildBottomSheetBarButton(AppColors.mainBackgroundColor, Icons.close, AppLocales.of(context).translate('actions.deselectAll'), onSelectAll)
+					else
+						buildBottomSheetBarButton(AppColors.mainBackgroundColor, Icons.playlist_add_check, AppLocales.of(context).translate('actions.selectAll'), onSelectAll),
+				*/		
```

Comment:
What about those comments?

Reply:
Prepared UI part for "Select all" functionality - we are not doing right now, but I already done the visuals and I think they are ok.